### PR TITLE
Doc - Explain i18n (client) limitation and a workaround

### PR DESCRIPTION
**Pull request by ger-benjamin**
Follow GEO-1767

Please verify that it's understandable.

### CODE REVIEW

File: doc/integrator/internationalization.rst [hunk 1]
```diff
@@ -71,6 +70,24 @@ For non Docker project:
      * some metadata as disclaimer
      * editable layer (database structure, data or enumerations)
 
+.. note::
+
+   In mapfiles, attributes added by the mapserver's substitution will be
```
Comment (jwkaltz) [inline doc/integrator/internationalization.rst]: In mapfiles, attributes added by mapserver substitution will not be collected for translation.

File: doc/integrator/internationalization.rst [hunk 2]
```diff
@@ -71,6 +70,24 @@ For non Docker project:
...
+   not collected for translation.
+
+~~~~~~~~~~~~~~~~~~~~~~~
+Collecte custom strings
```
Comment (jwkaltz) [inline doc/integrator/internationalization.rst]: Collect

File: doc/integrator/internationalization.rst [hunk 3]
```diff
@@ -71,6 +70,24 @@ For non Docker project:
...
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If the standard system can't collect some strings you can add them manually in
```
Comment (jwkaltz) [inline doc/integrator/internationalization.rst]: If the standard system can not collect some strings, you can add them manually in

File: doc/integrator/internationalization.rst [hunk 4]
```diff
@@ -71,6 +70,24 @@ For non Docker project:
...
+one of your JavaScript application controller:
```
Comment (jwkaltz) [inline doc/integrator/internationalization.rst]: controllers

**Comment by ger-benjamin** [comment]
@sbrunner can we merge (despite of travis) ? (I can't myself, I'm not admin.)

**Comment by ger-benjamin** [comment]
@sbrunner this branch is already up to date on 2.3. Please merge.

**Pull request merged**